pkg/webhook/validation.go

```diff
@@ -216,16 +216,6 @@ func (v *validator) validate(ctx context.Context, obj runtime.Object) (admission
 		}
 	}
 
-	path = field.NewPath("status")
-
-	conditionTypes := make(map[trustapi.BundleConditionType]struct{})
-	for i, condition := range bundle.Status.Conditions {
```

Comment:
Why are we removing this check?

Reply:
Please read the PR description. Do you think it's still needed? Any duplicates will be resolved by the map semantics on the list.

Reply:
I just noticed it when I had to change the map key type to `string`. But I can leave it if you insist. 😉 But I still think it's not required to have it.

Reply:
The change making this webhook validation obsolete was introduced in https://github.com/cert-manager/trust-manager/pull/173/files.